src/Twig/ContentExtension.php

```diff
@@ -215,4 +216,19 @@ public function getEditLink(Content $content, bool $absolute = false): ?string
             $absolute ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH
         );
     }
+
+    public function getTaxonomies(Content $content): Collection
+    {
+        $taxonomies = [];
+        foreach ($content->getTaxonomies() as $taxonomy) {
+            $link = $this->urlGenerator->generate('taxonomy', [
```

Comment:
why link needs to be pre-generated? it can be generated in Twig

Reply:
Using `{{ path('taxonomies', { … }) }}`, you mean? Yes it can, but this is nice for the endusers.